### ISSUE TITLE
MD060 was not triggered for aligned table without spaces in seperator line

### ISSUE THREAD

**Issue by thompson-tomo** [DavidAnson/markdownlint-cli2#718]
Even after updating to the latest version 0.19 of the package the following was not flagged as a violation.

```
| Stable Packages | Experimental Packages |
|-----------------|-----------------------|
| 2.0.x           | 0.200.x               |
```

I would've expected a violation based upon the examples in https://github.com/DavidAnson/markdownlint/blob/main/doc/md060.md showing the aligned table also having a single space separating | and -.

See https://github.com/open-telemetry/opentelemetry-js/pull/6118 for an example of it passing the pipeline.

**Comment by thompson-tomo** [comment]
Might be worthwhile to update the explanation as when you read the below and the examples

- Style aligned looks the most like a table:
- Style compact uses a single space to pad cell content:
- Style tight uses no padding for cell content:

You get the impression that aligned uses one or more   space to pad cell content and the content including header seperator/text is aligned just like the example.

Would the following examples also pass:

```
| Stable Packages | Experimental Packages |
|-----------------|-----------------------|
| 2.0.x           |0.200.x                |
|3.0.x            | 0.200.x               |
```

I don't think it needs a new rule but rather the style definition tightened up to ensure consistent padding for | like done for the other 2 styles.

**Comment by DavidAnson** [comment]
No need to ask, you can try it yourself! :) Indeed, because that is recognized as a table and the columns are aligned, no violations are produced:

https://dlaa.me/markdownlint/#%25m%23%20Issue%20718%0A%0A%7C%20Stable%20Packages%20%7C%20Experimental%20Packages%20%7C%0A%7C-----------------%7C-----------------------%7C%0A%7C%202.0.x%20%20%20%20%20%20%20%20%20%20%20%7C0.200.x%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7C%0A%7C3.0.x%20%20%20%20%20%20%20%20%20%20%20%20%7C%200.200.x%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7C%0A

**Comment by DavidAnson** [comment]
I've updated the documentation for the next release to call out that `MD060`'s `aligned` style is only looking at table cell dividers.

https://github.com/DavidAnson/markdownlint/commit/9b116ae889d475cafe1d57a98d43637ace8ba778
